contributors/design-proposals/resource-management/device-plugin.md

```diff
@@ -144,9 +136,9 @@ Devices reported by Device Plugins are advertised as Extended resources of
 the shape `vendor-domain/vendor-device`.
 E.g., Nvidia GPUs are advertised as `nvidia.com/gpu`
 
-Devices can be selected using the same process as for OIRs in the pod spec.
-Devices have no impact on QOS. However, for the alpha, we expect the request
-to have limits == requests.
+Devices can be selected using the same process as for OIRs in the container spec.
```

Comment:
OIR support has been removed from Kubernetes. Please use the new feature name "extended resources."

Reply:
Here and elsewhere in the doc.

Reply:
Consistency check: updated separately for website in this PR: https://github.com/kubernetes/website/pull/6501